constants/members.json

```diff
@@ -3412,9 +3420,9 @@
     {
       "name": "Oditha Amarasinghe",
       "role": "developer",
-      "term": 1235,
+      "term": 1236,
       "teams": ["casa"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FDev_Oditha_Amarasinghe.jpeg?alt=media&token=1dc5861c-2a97-45b2-bfaa-f79c26701446&_gl=1*k8k38z*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMjU5MS4wLjAuMA.."
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Oditha_Amarasinghe.jpeg?alt=media&token=7e71b36a-3275-4f8e-b52c-6c77d73833b0"
     },
     {
       "name": "Chen Kai Yan",
```

Comment:
I believe this term should be `1236`. They are still a developer for casa on the casa list